.lintstagedrc.js

```diff
@@ -1,9 +1,7 @@
+// NOTE: 'concurrent' had to be disabled to allow prettier to apply code changes before linting.
 module.exports = {
-  '*.{js,ts,vue}': [
-    () => 'vue-tsc --noEmit',
-    'eslint --config .eslintrc.fix.js --fix',
-  ],
-  '*.{css,vue,scss}': [
-    'stylelint --config .stylelintrc.fix.js --fix',
```

Comment:
Todo: we should keep `stylelint --fix` because it applies property sorting for CSS, which is not supported by Prettier...